src/components/Body.js

```diff
@@ -26,14 +26,17 @@ const Body = () => {
     );
 
     const json = await data.json();
+    const arrayOfCards = json.data.cards;
+    const restaurant_list = "restaurant_grid_listing";
```

Comment:
use camel case for naming variables along with optional chaining.

Reply:
Done